ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -381,71 +383,70 @@ def _filtered_wan_vrfs(self) -> list:
             wan_vrfs.append(
                 {
                     "name": "default",
-                    self._wan_policy_key: f"{self._default_vrf_policy['name']}-WITH-CP",
+                    "policy": f"{self._default_avt_policy['name']}-WITH-CP",
+                    "wan_vni": 1,
+                    "original_policy": self._default_avt_policy["name"],
```

Comment:
This is not used consistently to get the name of the default policy.

Reply:
can you pleaase point where this is not used?

Reply:
ok found what you meant looking at the code please check [f209b9b](https://github.com/aristanetworks/avd/pull/3600/commits/f209b9bd27c51a54a1499974eef39c6e7beb789d)

Reply:
You fixed the only one.